src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/model/FindResult.java

```diff
@@ -117,9 +315,22 @@ else if (openTags > 0)
       return ints;
    }
 
+   private String getJavaStringArray(String property)
+   {
+      JsArrayString array = getStringArray(property);
+      String string = array.toString();
+      return string;
+   }
+
    private native final JsArrayInteger getArray(String property) /*-{
-      if (this == null)
+      if (this === null)
```

Comment:
nit: while `===` is usually preferred for Javascript, using `==` here means that both `null` and `undefined` are properly handled. I'd suggest leaving this as `==`.

Reply:
If this[property] is undefined, we still return [] on the following line. I'm curious what @jmcphers  thinks since he requested this change originally.

Reply:
But if `this` itself is `undefined` then you will evaluate `undefined[property]` below, which will throw a JavaScript exception.

Reply:
Gotcha, fixed this.